beancount_import/source/schwab_csv.py

```diff
@@ -307,6 +321,38 @@ def get_processed_entry(
                 fees=self.fees,
                 **shared_attrs,
             )
+        if self.action is BrokerageAction.EXPIRED:
+            assert self.quantity is not None
+            amount = Amount(Decimal(0), currency=CASH_CURRENCY) if self.amount is None else self.amount
+            price = Decimal(0) if self.price is None else self.price
+            shared_attrs: SharedAttrsDict = dict(
+                capital_gains_account=capital_gains_account,
+                fees_account=fees_account,
+                account=account,
+                date=self.date,
+                action=self.action,
+                description=self.description,
+                quantity=self.quantity,
+                symbol=self.symbol,
+                price=price,
+                fees=self.fees,
+                amount=amount,
+                filename=self.filename,
+                line=self.line,
+            )
```

Comment:
rather than duplicating the creation of the shared_attrs dict, let's follow the existing pattern of `STOCK_PLAN_ACTIVITY` which also needs to do some prior munging of `amount`, and have two `if self.action is BrokerageAction.EXPIRED`, one before the construction of `shared_attrs` and one after.

Reply:
Done.